Resources/Prototypes/Entities/Objects/Fun/toys.yml

```
@@ -809,6 +811,14 @@
     wideAnimationRotation: 180
     soundHit:
       path: /Audio/Items/Toys/rattle.ogg
+  - type: Clothing
+    slots:
+    - NECK
+    quickEquip: false
+    clothingVisuals:
+      neck:
+      - state: plushiesnake-equipped-NECK
+        offset: "0, 0"
```

clarity, I also added the speech component with the reptilian accent cause snake
```suggestion
  - type: Clothing
    quickEquip: false
    equippedPrefix: plushiesnake
    slots:
    - NECK
  - type: Speech
    speechVerb: Reptilian # for pais (In the secret stash)
```